### PR TITLE
Add support to turn on/off tracing for an instance

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#459

### CODE REVIEW

**Comment by sam-github** [comment]
it's pretty useful to be able to turn on tracing... though I guess, you can still do `STRONGLOOP_TRACING=on sl-pm`, and tracing will be auto-enabled, right?

**Comment by kraman** [comment]
@sam-github I can add back `--trace` on PM... but that would turn on tracing on all processes on this PM, not just one service....

**Comment by kraman** [comment]
Removed the `--trace` command line flag but can still turn on tracing on all services by setting env `STRONGLOOP_TRACING=on`

File: lib/drivers/direct/container.js [hunk 1]
```diff
@@ -101,6 +101,13 @@ Container.prototype.setStartOptions = function(options) {
       self.debug('set-size %j: rsp %j', options.size, rsp);
     });
   }
+
```
Comment (sam-github) [inline lib/drivers/direct/container.js]: I'm OK to LGTM this for now, it unblocks QA, but we need to modify mesh-models to only update if there are changes.

Also, this code will need to be copied into the docker driver.

**Comment by sam-github** [comment]
I think tracing is only supported for direct driver, not the docker.

Other than that, LGTM

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by sam-github** [comment]
LGTM

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
tests passed, squashing/merging

**Pull request merged**